### DISCUSSION TITLE
3.11: unable to delete a stream

### DISCUSSION THREAD

**Discussion by CHAITANYAMALINENI** [rabbitmq/rabbitmq-server#11960] [Questions]
I tried creating rabbitmq stream type queue through management console in a 3 node cluster and queue creation got stuck and it was in bad state. Later on, whenever I click on queues tab we are getting 500 internal server error and queues page is not showing up. I am attaching screenshot for reference. 

The new stream queue that was created is showing up in list using "rabbitmqctl list_queues" command and it was in bad state.
I tried to delete queue using "rabbitmqctl delete" command thinking that it might resolve issue but I am getting queue not found error. Could someone please help resolve this issue. Thanks.

RabbitMq version: 3.11.28
Erlang version: 25.3.2.9

![Screenshot 2024-08-08 at 3 52 08 PM](https://github.com/user-attachments/assets/5d9c6cf5-5e5b-44fd-8c80-15534bb83d28)

**Comment by michaelklishin** [comment]
RabbitMQ 3.11.x [has reached EOL](https://www.rabbitmq.com/release-information) (even for paying customers).

Please [upgrade](https://www.rabbitmq.com/docs/upgrade).
